jujugui/static/gui/src/app/components/search-results/search-results.js

```diff
@@ -253,7 +253,7 @@ YUI.add('search-results', function(Y) {
     },
 
     render: function() {
-      var html = Handlebars.templates['search-results.hbs'](this.state.data);
+      var html = Handlebars.templates['search-results-tmpl.hbs'](this.state.data);
```

Comment:
Name change is necessary because YUI was getting confused about two search-results.js files, once the template was compiled.